### PR TITLE
fix(bundler): Set exec cwd to lock file

### PR DESCRIPTION
**Pull request by jorge-wonolo**
## Changes

Execute bundler commands for artifacts based on lock file path.

## Context

Given that Gemfile.lock is in the root directory of the project and Gemfile or Gemfiles are in subdirectories (e.g.: `teamA/Gemfile`) then bundler would have been executed under `./teamA` path missing the root folder lock file and therefore only updating the Gemfile and not the lock file

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [X] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [X] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

Tested against: https://github.com/jorge-wonolo/renovate-test/pull/12

### CODE REVIEW

**Review by daande — APPROVED** [review]
lg2m

File: lib/modules/manager/bundler/artifacts.ts [hunk 1]
```diff
@@ -77,8 +77,8 @@ export async function updateArtifacts(
     logger.debug('Aborting Bundler artifacts due to previous failed attempt');
     throw new Error(existingError);
   }
-  const lockFileName = await getLockFilePath(packageFileName);
```
Comment (rarkins) [inline lib/modules/manager/bundler/artifacts.ts:80]: Why is this variable rename important? It seems unrelated to the fix?

**Review by viceice — DISMISSED** [review]
probably needs test changes

**Comment by jorge-wonolo** [comment]
> probably needs test changes

@viceice added a test, thanks for the suggestion

**Review by viceice — APPROVED** [review]
